framework/src/executioners/Transient.C

```diff
@@ -474,6 +474,8 @@ Transient::endStep(Real input_time)
 
   _picard_converged=false;
 
+  bool ts_output =_time_stepper->enableOutput();
```

Comment:
@permcody .  The reason I moved the call to this function is because I'm doubling the purpose of it.  I'm also using it as a kind of an endStep() for my time stepper.  I need Transient to call a function after the picard iterations and before the output.  So I put some stuff in there expecting it gets called at that location in Transient. I hope that makes sense.